synthetics-stats/src/entities/userBalance.ts

```diff
@@ -0,0 +1,127 @@
+import { BigInt, log } from "@graphprotocol/graph-ts";
+import {
+  CollectedMarketFeesInfo,
+  LatestUserGmTokensBalanceChangeRef,
+  Transaction,
+  UserGmTokensBalanceChange
+} from "../../generated/schema";
+import { getOrCreateCollectedMarketFees } from "./fees";
+
+let ZERO = BigInt.fromI32(0);
+
+export function saveUserGmTokensBalanceChange(
+  account: string,
+  marketAddress: string,
+  value: BigInt,
+  transaction: Transaction,
+  transactionLogIndex: BigInt,
+  postfix: string
```

Comment:
can we use something like `isDeposit` or `actionType`? `postfix` with values in|out is a bit non-intuitive

Reply:
or actually derive it from `value` as you already do on line 26